schemas/1.21.0

```diff
@@ -50,6 +50,33 @@ versions:
         # https://github.com/open-telemetry/semantic-conventions/pull/53
         - rename_metrics:
             process.runtime.jvm.cpu.utilization: process.runtime.jvm.cpu.recent_utilization
+        # https://github.com/open-telemetry/opentelemetry-specification/pull/3464
+        - rename_attributes:
+            attribute_map:
+              memory: memory.type
```

Comment:
This change seems to be missing still. It's still just `type` in the rendered markdown.

Reply:
thanks for catching - this got lost rebasing